app/controllers/v0/in_progress_forms_controller.rb

```diff
@@ -19,10 +21,28 @@ def update
              InProgressForm.new(form_id:, user_uuid: @current_user.uuid)
       form.user_account = @current_user.user_account
       form.real_user_uuid = @current_user.uuid
+
+      log_cfi_metric(form)
+
       form.update!(form_data: params[:form_data] || params[:formData], metadata: params[:metadata])
       render json: form, key_transform: :unaltered
     end
 
+    def log_cfi_metric(form)
+      if form.metadata['cfiMetric'].blank?
+        fd = params[:form_data] || params[:form_Data]
+        fd = JSON.parse(fd) if fd.is_a?(String)
+        if fd.present? &&
+           (fd.dig('view:claim_type', 'view:claiming_increase') || fd.dig('view:claimType', 'view:claimingIncrease'))
+          max_cfi_enabled = Flipper.enabled?(:disability_526_maximum_rating) ? 'on' : 'off'
+          StatsD.increment("#{MAX_CFI_STATSD_KEY_PREFIX}.#{max_cfi_enabled}.rated-disabilities")
+          params[:metadata]['cfiMetric'] = true
+        end
+      else
+        params[:metadata]['cfiMetric'] = true
```

Comment:
I don't actually think you want this else clause, because it'll mark metadata.cfiMetric as true the first time the IPF gets saved. Shouldn't it only get marked true when the StatsD inc gets fired?

This might be a good thing to write a contrived unit test for.